### DISCUSSION TITLE
Suggestion for Ko-Fi Support Link - Let People know it's in AUD

### DISCUSSION THREAD

**Discussion by ziptbm** [bezmi/hass_custom_components#23] [General]
Really appreciate this custom component and wish it were part of the core HA experience.  I sent in a small donation to show my appreciation, but there was nothing on Ko-Fi to indicate the currency was in AUD.  Just making a quick suggestion to let others know so they can take that into consideration. @bezmi

**Comment by bezmi** [comment]
Thanks for the donation! I think we are a fair ways from the level of quality expected from a core HA integration, but hopefully with the web UI configuration/settings it'll be one step closer.

It's strange that Ko-fi doesn't show the currency, I only set it up recently. Perhaps I'll just set it to USD to avoid the ambiguity.
